I/isoband/build_tarballs.jl

```diff
@@ -0,0 +1,34 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "isoband"
+version = v"0.2.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/jkrumbiegel/isoband.git", "8c0050541c02d0cc5b3dd737c78cbf81ee61f802")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir ${libdir}
+$CXX -shared -std=c++11 -O3 -fPIC -o ${libdir}/isoband isoband/src/isoband.cpp
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("isoband", :isoband)
```

Comment:
To match the name of the file I suggested above:
```suggestion
    LibraryProduct("libisoband", :libisoband)
```